test/sanity/ignore.txt

```diff
@@ -1135,19 +1131,19 @@ lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine.py validate-mo
 lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualnetwork.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py validate-modules:doc-missing-type
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset_facts.py validate-modules:parameter-type-not-in-doc
+lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset_info.py validate-modules:parameter-type-not-in-doc
```

Comment:
These changes are unrelated to the PR. Perhaps there was an issue rebasing?

Reply:
Yeah probably. I'll fix that now.

Reply:
Fixed